docs/GraFx-Studio/convert/Adobe-InDesign/index.md

```diff
@@ -0,0 +1,196 @@
+2. **Ignore**: The preflight engine will change the missing feature to a supported version (e.g. stroke type in the example).
+3. **Fix the Issue**: an obvious but not mentioned choice, is you choose to fix it by choosing a different feature in Adobe® InDesign®, and re-run the preflight.
+
+![screenshot-full](convert19.png)
+
+!!! info "Placed Assets: Pros and Cons"
+
+    **Benefits**  
+    - **Preserves Quality**: When elements are saved as PDF assets, their original quality and appearance remain intact.
+    - **Simplifies Conversion**: For complex designs, converting to a PDF asset can streamline the process by avoiding the need for manual adjustments.  
+    
+    **Limitations**  
+    - **Not Editable**: Placed assets are static. You cannot edit them within GraFx Studio. For example, if text is saved as a PDF asset, you won’t be able to create a text variable to dynamically alter the text content.
+  
+
+## Supported Features
+
+As the GraFx Studio Exporter is **Experimental**, the list below will update frequently.
+
+### Feature Support Table
+
+Features marked with a green checkmark ✅ are fully supported. Some unsupported features are listed for clarity where it matters most.
+If a feature isn’t mentioned, it’s not supported — for now! Stay tuned, as we’re continually expanding support.
+
+| **Category**           | **Feature**                              | **Support Level**        | **Notes**                                           |
+|-------------------------|------------------------------------------|--------------------------|----------------------------------------------------|
+| **Plugin UI**          | Selecting a folder for export            | ✅             |                                                    |
+|                        | Selecting a page of the document to export | ✅            |                                                    |
+|                        | Logging                                   | ✅             |                                                    |
+|                        | Appearance color scheme                  | ✅             |                                                    |
+| **Document**           | Page size                                | ✅             |                                                    |
+|                        | Choose the page to import                                | ✅            | 
+|                        | All pages                                | ❌           |                                      |
+|                        | Layouts                                  | ❌           |                                      |
+| **Frames**             | Rotation                                 | ✅             |                                                    |
+|                        | Blend modes                              | ❌           |                                      |
```

Comment:
I think this is supported. Or at least partially.

Reply:
Blend modes are not enabled in Plugin. So for now this option is not supported. But it is in our 'soon' list